src/lib.rs

```diff
@@ -540,26 +540,28 @@ where
 #[macro_export]
 macro_rules! query_group {
     (
-        $(#[$attr:meta])* $v:vis trait $name:ident { $($t:tt)* }
+        $(#[$attr:meta])* $v:vis trait $name:ident $(<$lt:lifetime>)* { $($t:tt)* }
```

Comment:
I wonder if, while we're at it, we want to support other sorts of parameters? Regardless, this should probably be something like `$(<$($lt:lifetime,)*>)?`, so that one can use the syntax `<'a, 'b>` (I forget, is `?` stable yet..?)